common_test.go

```diff
@@ -4,22 +4,54 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"os/exec"
 	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/suite"
 )
 
-type SimpleLinguistTestSuite struct {
+const lingustURL = "https://github.com/github/linguist.git"
+
+type EnryTestSuite struct {
 	suite.Suite
+	tmpLinguist string
+}
+
+func TestEnryTestSuite(t *testing.T) {
+	suite.Run(t, new(EnryTestSuite))
+}
+
+func (e *EnryTestSuite) SetupSuite() {
+	tmpLinguist, err := ioutil.TempDir("", "linguist-")
+	assert.NoError(e.T(), err)
+	e.tmpLinguist = tmpLinguist
+
+	cmd := exec.Command("git", "clone", lingustURL, tmpLinguist)
```

Comment:
I mean, can we just use a very small and representative corpus as a fixture, generate whatever you want to generate from it and test it, instead of downloading a whole repo to a local dir and then checking out a specific version of that repo?